src/home/index.js

```diff
@@ -1,9 +1,34 @@
 import React from 'react'
+import GoogleLogin from 'react-google-login'
+import postTokenToServer from '../lib/auth'
 
-export const Home = () => (
-	<div>
-		<p> Welcome to Oleville Voting! </p>
-	</div>
-)
+class Home extends React.Component {
+
+	googleResponseSuccess(response) {
+		postTokenToServer(response.tokenObj.id_token)
+	}
+
+	googleResponseFailure(response) {
+		console.log(response)
+	}
+
+	render() {
+		return (
+				<div>
+					<p> Welcome to Oleville Voting! </p>
+					<p> Please log in to access your ballot.</p>
+					<GoogleLogin
+						clientId='437117106108-miaqebkeantfo00063nottgvsd419ns9.apps.googleusercontent.com'
```

Comment:
Should we publish this?

Reply:
We have to. It's not meant to be secret, there's a separate _clientSecret_ for that. This is to ensure that the token that the user sends us is a token meant for our app. When we do the verification step with google, they send us the clientId that generated the token and we check it against our clientId. If they don't match, then the login process was tampered with and we reject the token, preventing issues with this going out to the user.